charts/generic/k8-prometheus-operator/engine.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: k8-prometheus-operator
+  namespace: default
+spec:
+  appinfo:
+    appns: 'default'
+    applabel: "k8s-app=prometheus-operator"
+    appkind: deployment
+  annotationCheck: 'false'
+  engineState: 'active'
+  chaosServiceAccount: chaos-admin
+  monitoring: false
+  jobCleanUpPolicy: 'retain'
+  experiments:
+    - name: k8-pod-delete
+      spec:
+        components:
+          env:
+            # set chaos namespace
+            - name: NAME_SPACE
```

Comment:
do mention in comment .. the namespace could be this or something else .. how you implemented prometheus